src/Button.cpp

```diff
@@ -59,6 +59,8 @@ static void Button_DoButtonActions(void);
 void Button_Init() {
     #if (WAKEUP_BUTTON <= 39)
         esp_sleep_enable_ext0_wakeup((gpio_num_t)WAKEUP_BUTTON, 0);
+        esp_sleep_enable_ext0_wakeup((gpio_num_t)ROTARYENCODER_CLK, 0);
```

Comment:
you need to check #ifdef USEROTARY_ENABLE here for Line 62 and 63

Reply:
Thank's for reviewing!
OK, this change was wrong, reverted.